keel-core/src/main/kotlin/com/netflix/spinnaker/keel/exceptions/InvalidVetoException.kt

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spinnaker.keel.exceptions
+
+import org.omg.CORBA.UserException
+
+class InvalidVetoException(application: String, environment: String, reference: String, version: String) :
+  UserException("Unable to veto artifact version $version with reference $reference in environment " +
+  "$environment and application $application. Is this version pinned?")
```

Comment:
Is pinning the only reason to get this exception?

Reply:
I think there might be another way (like something fails?) but this is the most probable way, and as far as I could tell the only way the 'veto' function would return false (false == veto didn't work). That's why I kept this like asking a question